packages/core/src/shader/Program.ts

```diff
@@ -47,8 +42,15 @@ export class Program
     nameCache: any;
     glPrograms: { [ key: number ]: GLProgram};
     syncUniforms: any;
+    /**
+     * assigned when a program is first bound to the shader system
+     */
```

Comment:
More terser comment works too!
```suggestion
    /** Assigned when a program is first bound to the shader system. */
```